src/functions-reference/unbounded_discrete_distributions.Rmd

```diff
@@ -363,24 +407,64 @@ dropping constant additive terms.
 
 ### Stan Functions
 
+<!-- real; poisson_log_glm_lpmf; (int y | matrix x, real alpha, vector beta); -->
+\index{{\tt \bfseries poisson\_log\_glm\_lpmf }!{\tt (int y \textbar\ matrix x, real alpha, vector beta): real}|hyperpage}
+
+`real` **`poisson_log_glm_lpmf`**`(int y | matrix x, real alpha, vector beta)`<br>\newline
+The log Poisson probability mass of y given log-rate `alpha + x * beta`,
+where a constant intercept `alpha` and dependent variable value `y` are used for all observations. 
+The number of columns of `x` needs to match the length of the weight vector `beta`.
+If `x` and `y` are data (not parameters) this function can be executed on a GPU.
+
+<!-- real; poisson_log_glm_lpmf; (int y | matrix x, vector alpha, vector beta); -->
+\index{{\tt \bfseries poisson\_log\_glm\_lpmf }!{\tt (int y \textbar\ matrix x, vector alpha, vector beta): real}|hyperpage}
+
+`real` **`poisson_log_glm_lpmf`**`(int y | matrix x, vector alpha, vector beta)`<br>\newline
+The log Poisson probability mass of y given log-rate `alpha + x * beta`,
+where an intercept `alpha` is used that is allowed to vary with the
+different observations. Dependent variable value `y` is used for all observations. 
+The number of rows if `x` must match the length of `alpha` and 
```

Comment:
rows if `x` -> rows of `x`
length -> size (here and everywhere else)
weight -> coefficient or parameter (also everywhere)

Reply:
fixed